sshuttle/ssnet.py

```diff
@@ -243,7 +243,9 @@ def fill(self):
     def copy_to(self, outwrap):
         if self.buf and self.buf[0]:
             wrote = outwrap.write(self.buf[0])
-            if wrote > 0:
+            if wrote is None:
+                log("ssnet.py copy_to 'wrote' is None\n")
+            elif wrote > 0:
                 self.total_wrote += wrote
             self.buf[0] = self.buf[0][wrote:]
```

Comment:
perhaps this line should be indented under the elif in case `wrote` is `None`. Alternatively you can set wrote to 0 if it is None if that makes sense with the logic here.

Reply:
ya that might cause a problem for python 3. In 2.7 it converts a None to 0, so this might have just moved the error down a few lines

Reply:
before delivering the change, could you please sanity test the fix just to be sure. thanks